x-pack/plugins/maps/public/classes/layers/new_vector_layer_wizard/config.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import { LayerWizard, RenderWizardArguments } from '../../layers/layer_wizard_registry';
+import { NewVectorLayerEditor } from './index';
+import { DrawLayerIcon } from '../../layers/icons/draw_layer_icon';
+import { getFileUpload } from '../../../kibana_services';
+
+const ADD_VECTOR_DRAWING_LAYER = 'ADD_VECTOR_DRAWING_LAYER';
+
+export const newVectorLayerWizardConfig: LayerWizard = {
+  categories: [],
```

Comment:
Category should be LAYER_WIZARD_CATEGORY.ELASTICSEARCH so it shows up under the elasticsearch filter.